lib/include/jxl/types.h

```diff
@@ -115,6 +115,30 @@ typedef struct {
  */
 typedef char JxlBoxType[4];
 
+/** Types of progressive detail
+ * Currently only the following level of progressive detail is implemented:
+ *  - kDC (which implies kFrames)
+ *  - kPasses (which implies kLastPasses, kDC and kFrames)
```

Comment:
Maybe just write that every N implies events for 0..N?

Reply:
done